web/satellite/src/components/accessGrants/AccessGrants.vue

```diff
@@ -1,8 +1,8 @@
-// Copyright (C) 2020 Storj Labs, Inc.
```

Comment:
nit - if this file was originally created in 2020, you don't need to update the copyright header
if you added the file recently, it should be 2022

Reply:
This appears as 2022 on the version I just pulled. Unsure why it is showing differently on your end.